config/frontend.example.toml

```diff
@@ -35,10 +35,10 @@ body_limit = "64MB"
 ## The gRPC server options.
 [grpc]
 ## The address to bind the gRPC server.
-addr = "127.0.0.1:4001"
-## The hostname advertised to the metasrv,
+bind_addr = "127.0.0.1:4001"
+## The address advertised to the metasrv,
 ## and used for connections from outside the host
-hostname = "127.0.0.1:4001"
+server_addr = "127.0.0.1:4001"
```

Comment:
Sugg: 
- About default value for `server_addr`, like "default to bind_addr"
- Describe the intended use case

Reply:
resolved in 736428091b45999d8b6f75f838a3eebd7b7c127e